src/services/ledger/index.types.d.ts

```diff
@@ -36,29 +36,30 @@ type AddOnchainSendLedgerMetadata = NonIntraledgerLedgerMetadata & {
 type AddColdStorageLedgerMetadata = NonIntraledgerLedgerMetadata & {
   hash: OnChainTxHash
   payee_addresses: OnChainAddress[]
-  currency: WalletCurrency
+  currency?: WalletCurrency
```

Comment:
Why did this become optional?

Reply:
Not sure how that happened, I will remove that.